### PR TITLE
cmake, test: enable ceph-disk.sh to run with cmake

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
@dachary and @alimaredia could you take a look? thanks.

**Comment by dachary** [comment]
<pre>
Reviewed-by: Loic Dachary &lt;ldachary@redhat.com>
</pre>

as soon as it passes make check

File: src/ceph-disk/tests/ceph-disk.sh [hunk 1]
```diff
@@ -339,50 +304,78 @@ function test_activate_dir_bluestore() {
...
-    test_activate_dir 2>&1 | tee $DIR/test_find
-    ! grep "No cluster conf found in $DIR" $DIR/test_find || return 1
-    teardown
-
-    setup
-    rm $DIR/ceph.conf
-    test_activate_dir > $DIR/test_find 2>&1 
-    grep --quiet "No cluster conf found in $DIR" $DIR/test_find || return 1
-    teardown
+    local dir=$1
```
Comment (tchaikov) [inline src/ceph-disk/tests/ceph-disk.sh]: the setup and teardown here are removed as `run()` calls them for us already.

**Comment by tchaikov** [comment]
"make check" timed out:
```
make[5]: *** [test/cephtool-test-mds.sh.log] Terminated
```

and  ceph-disk.sh passed:

```
PASS: ceph-disk/run-tox.sh
```

**Pull request merged**